res/values-fr/strings.xml

```diff
@@ -41,7 +41,7 @@
   <string name="prefs_remember_last_share_location">Mémoriser l\'emplacement de partage</string>
   <string name="prefs_remember_last_upload_location_summary">Mémoriser le dernier emplacement d\'upload</string>
   <string name="recommend_subject">Essayez %1$s sur votre smartphone&amp;nbsp;!</string>
-  <string name="recommend_text">J'aimerais vous inviter à utiliser %1$s sur votre smartphone !
+  <string name="recommend_text">J\'aimerais vous inviter à utiliser %1$s sur votre smartphone !
```

Comment:
Right. This was because the wrong translation kept me from compiling.
But I should not have push this change.
I will revert this.